### PR TITLE
fix(deps): Replace cli-color dep with colorette

### LINKED ISSUE #56: [bug] dependency `cli-color` has licensing issues
According to Socket, [`cli-color` has issues](https://socket.dev/npm/package/cli-color/issues/2.0.3) related to licensing. We are currently using the MIT license for the packages we publish on npm and intend to continue doing so. This is kind of a hard requirement since a lot of projects have a strict requirement of having an MIT license. It is also important that the dependencies of our dependencies are MIT licensed as well.

We may be able to swap this one out for [`colorette`](https://www.npmjs.com/package/colorette), which is MIT-licensed ([silver quality](https://socket.dev/npm/package/colorette)) and maintained by my good friend @jorgebucaran. It has no dependencies.

Refs: https://github.com/DerekNonGeneric/loader339/pull/11#discussion_r478740474

### PR DESCRIPTION
**Pull request by dadolhay**
# What this PR do?

- Replaces the `cli-color` dependency with `colorette`

closes #56

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@
 // -----------------------------------------------------------------------------
 
 import { format } from 'util';
-import clc from 'cli-color';
+import * as colorette from "colorette";
```
Comment (DerekNonGeneric) [inline index.ts:8]:
```suggestion
import * as colorette from 'colorette';
```

We have been using single quotes for this file (and throughout much of the codebase); let's continue doing that.

File: index.ts [hunk 2]
```diff
@@ -5,7 +5,13 @@
...
 
 import { format } from 'util';
-import clc from 'cli-color';
+import {
+  blue as coloretteBlue,
+  italic as coloretteItalic,
+  red as coloretteRed,
+  underline as coloretteUnderline,
+  yellow as coloretteYellow
+} from "colorette";
```
Comment (DerekNonGeneric) [inline index.ts:14]:
```suggestion
} from 'colorette';
```

**Review by DerekNonGeneric — APPROVED** [review]
Looks good to me! I wonder if @jorgebucaran thinks so too. :)

File: index.ts [hunk 3]
```diff
@@ -5,7 +5,13 @@
 // -----------------------------------------------------------------------------
 
 import { format } from 'util';
-import clc from 'cli-color';
+import {
+  blue as coloretteBlue,
+  italic as coloretteItalic,
+  red as coloretteRed,
+  underline as coloretteUnderline,
+  yellow as coloretteYellow
```
Comment (DerekNonGeneric) [inline index.ts:13]:
```suggestion
  yellow as coloretteYellow,
```

A trailing comma here is preferable, apparently.

Comment (dadolhay) [reply index.ts]: Added it

**Comment by dadolhay** [comment]
@DerekNonGeneric it seems, that it does not agree with your change about the import. What do you want me to do? leave it as is, and you'll ignore the warning, or rewrite all imports to double quotes so it's consistent and passes the test?

**Review by jorgebucaran — APPROVED** [review]
Looks great to me!

**Comment by DerekNonGeneric** [comment]
> leave it as is, and you'll ignore the warning

Yeah, that's what we'll do here; apparently it needs to be reconfigured &mdash; not a problem!

**Pull request merged**